homeassistant/components/rfxtrx/__init__.py

```diff
@@ -211,6 +208,14 @@ def _add_device(event, device_id):
         config = {}
         config[CONF_DEVICE_ID] = device_id
 
+        _LOGGER.info(
```

Comment:
```suggestion
        _LOGGER.debug(
```

Looks like this is new. I'd probably go with debug unless there is a strong use case for the user knowing this

Reply:
I'd say there is good reason. A persistent notification might be even better. Then again not all added devices are owned by the users, so might be too much.